app/views/publish/courses/confirmation.html.erb

```diff
@@ -1,6 +1,10 @@
 <% page_title = "Check your answers" %>
 <% content_for :page_title, title_with_error_prefix(page_title, course.errors.any?) %>
 
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(request.referer) %>
```

Comment:
Doesn't quite work

https://publish-teacher-training-pr-3366.london.cloudapps.digital/publish/organisations/T92/2023/courses/confirmation?course%5Bage_range_in_years%5D=7_to_11&course%5Bapplications_open_from%5D=2022-10-11&course%5Bcampaign_name%5D=&course%5Bcan_sponsor_student_visa%5D=false&course%5Bfunding_type%5D=fee&course%5Bis_send%5D=0&course%5Blevel%5D=primary&course%5Bmaster_subject_id%5D=7&course%5Bqualification%5D=pgde_with_qts&course%5Bsites_ids%5D%5B%5D=11237735&course%5Bstart_date%5D=September+2023&course%5Bstudy_mode%5D=full_time_or_part_time&course%5Bsubjects_ids%5D%5B%5D=7

You want to regenerate the url

https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/models/concerns/publish/course_basic_detail_concern.rb#L201-L202

Reply:
Forgot about that 🙈